text_mining.py

```diff
@@ -0,0 +1,131 @@
+import requests
+import nltk
+import operator
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from collections import Counter
+
+
+papers = ['fedpaper1.txt', 'fedpaper2.txt', 'fedpaper3.txt', 'fedpaper4.txt', 'fedpaper5.txt', 'fedpaper6.txt', 'fedpaper7.txt', 'fedpaper8.txt', 'fedpaper9.txt', 'fedpaper10.txt']
+
+
+def fed_paper_creator():
+    '''
+    This function writes the lines from each one of the first 10 Federalist Papers to a corresponding
+    text file. It makes the words into one string, and makes them all lowercase
+    '''
+    input_file = open("fedpapers.txt", "r")
+    lines = input_file.readlines()
+
+    paper1 = str.lower(' '.join(lines[32:194]))
```

Comment:
What is the significance of the indices here? (ex. `32` and `194` in `lines[32:194]`). What these numbers represent would be a good thing to write in an in-line comment.